packages/portal/src/components/search/SearchQueryBuilder.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <b-row>
+    <div
+      v-for="(rule, index) in queryRules"
+      :key="index"
+      class="d-flex align-items-center"
+    >
+      <SearchQueryBuilderRule
+        :search-term="rule.searchTerm"
+        :selected-field="rule.selectedField"
+        :selected-modifier="rule.selectedModifier"
+        :search-fields="searchFields"
+        :modifiers="modifiers"
+        @change="(field, value) => rule[field] = value"
+      />
+      <b-button
+        :disabled="disableClearRuleButton"
+        @click="clearRule(index)"
+      >
+        {{ $t('search.advanced.actions.clear') }}
+      </b-button>
+    </div>
+    <b-button @click="searchWithBuildQueries">
+      {{ $t('search.advanced.actions.search') }}
+    </b-button>
+    <b-button @click="addNewRule">
+      {{ $t('search.advanced.actions.add') }}
+    </b-button>
+  </b-row>
+</template>
+
+<script>
+  import SearchQueryBuilderRule from './SearchQueryBuilderRule.vue';
+  export default {
+    name: 'SearchQueryBuilder',
+
+    components: {
+      SearchQueryBuilderRule
+    },
+
+    data() {
+      return {
+        queryRules: [{}],
+        searchFields: [
```

Comment:
If these are always the same, does it make sense for them to be here, and a prop on SearchQueryBuilderRule, or should they just be static local data in that component?

Reply:
Will be done on other branch: https://github.com/europeana/portal.js/tree/feat/EC-6313-searchQueryBuilder-parsing